src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/TestEnvironment.cs

```diff
@@ -234,15 +233,18 @@ private void Login()
             string password = this.ConnectionString.GetValue(ConnectionStringKeys.PasswordKey);
             string spnClientId = this.ConnectionString.GetValue(ConnectionStringKeys.ServicePrincipalKey);
             string spnSecret = this.ConnectionString.GetValue(ConnectionStringKeys.ServicePrincipalSecretKey);
-
-            /*
-            Currently we prioritize login as below:
-            1) UserName / Password combination
-            2) ServicePrincipal/ServicePrincipal Secret Key
-            3) Interactive Login (where user will be presented with prompt to login)
-           */
-            #region Login
-            ActiveDirectoryServiceSettings aadServiceSettings = new ActiveDirectoryServiceSettings()
+            //We use this because when login silently using userTokenProvider, we need to provide a well known ClientId for an app that has delegating permissions.
+            //All first party app have that permissions, so we use PowerShell app ClientId
+            string PowerShellClientId = "1950a258-227b-4e31-a9cf-717495945fc2";
```

Comment:
@shahabhijeet I'm not sure if this is sufficient to solve the problem, at least for the Search tests. Before these changes, this was the client ID being used, but the Tenant ID was blank. I don't think this client ID works with the new default Tenant ID, at least for the account we use for Search tests.